ygen/enumgen.go

```diff
@@ -440,103 +562,132 @@ func findEnumSet(entries map[string]*yang.Entry, compressPaths, noUnderscores, s
 type enumGenState struct {
 	// definedEnums keeps track of generated enum names to avoid conflicts.
 	definedEnums map[string]bool
-	enumSet      *enumSet
+	// enumSet contains the final, deduped generated enumerated value names
+	// to be returned by findEnumSet.
+	enumSet *enumSet
+	// enumeratedTypedefDedupSets stores the set of unique string keys
```

Comment:
Here's where I think that the "name clash" terminology makes more sense -- since 'deduplication' has typically mean that two leaves with *different* paths share the same underlying type.